include/Reader/abi.h

```diff
@@ -168,6 +168,9 @@ class ABIInfo {
                        ABIType ResultType, llvm::ArrayRef<ABIType> ArgTypes,
                        ABIArgInfo &ResultInfo,
                        std::vector<ABIArgInfo> &ArgInfos) const = 0;
+
+  /// \brief Virtual Destructor
+  virtual ~ABIInfo()=default;
```

Comment:
To avoid warnings from [clang-diagnostic-delete-non-virtual-dtor]
warning: delete called on 'ABIInfo' that is abstract but has non-virtual destructor